src/plasma/plasma_manager.c

```diff
@@ -1114,9 +1168,10 @@ void request_fetch_or_status(object_id object_id,
     /* Wait for the object data for the default number of retries, which timeout
      * after a default interval. */
     object_req->num_retries = NUM_RETRIES;
-    object_req->timer =
-        event_loop_add_timer(client_conn->manager_state->loop, MANAGER_TIMEOUT,
-                             fetch_timeout_handler, object_req);
+    object_req->timer = event_loop_add_timer(client_conn->manager_state->loop,
+                                             MANAGER_TIMEOUT,
+                                             fetch_timeout_handler,
+                                             object_req);
     request_transfer_from(client_conn, object_id);
     /* let scheduling the fetch request proceded and return */
   }
```

Comment:
the PLASMA_OBJECT_REMOTE message after this line can only be sent if manager_count is not zero